capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -68,10 +73,84 @@ export class SearchUserModal extends Component {
       }, 500)
     })
   }
+  
+  componentDidMount() {
+    this.fetchCollaborators();
+  }
+
+  fetchCollaborators = async () => {
+
+    if (!(this.props.type === 'booklists' || this.props.bookList)) {
+      return;
+    }
+
+    else if (this.props.bookList.collaboratorsIDs.length === 0) {
+      return;
+    }
+
+    let collaborators = [];
+
+    await Promise.all(this.props.bookList.collaboratorsIDs.map(async (collaboratorId) => {
+
+      const collaborator = await fetch(`/api/user?id=${collaboratorId}`).then(resp => resp.json());
+
+      delete collaborator.tokenObj;
```

Comment:
Smart here to delete the token object from getting added to the collaborators list, but is this tokenObject for each collaborator still visible from the user perspective on the network tab?
Is there a simple way to avoid this or would it require a large backend change